common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -505,4 +505,14 @@ trait FeatureSwitches {
     sellByDate = never,
     exposeClientSide = false
   )
+
+  val HeaderFeedback = Switch(
+    SwitchGroup.Feature,
+    "header-feedback",
+    "When ON a feedback prompt will be visible within the header test",
+    owners = Seq(Owner.withGithub("zeftilldeath")),
+    safeState = Off,
+    sellByDate = new LocalDate(2017, 8, 17),
```

Comment:
`2017, 7, 18` would be better I think.

Mostly because July, and Tuesdays are better than Mondays